landing-page/landing-page.html

```diff
@@ -0,0 +1,15 @@
+<html>
+    <head>
+        <style>
+        body {
+            background-color: white;
+        }
+        </style>
+    </head>
+    <body>
+        <h1 style="text-align:center;">Gizmatic</h1>
+        <img src="http://www.tapir.caltech.edu/~phopkins/Site/_Media/droppedimage-2_med_hr_med_hr.jpeg"></img>
+        <p>About us:</p>
+        <p>Founded in 1901 Gizmatic has been making the world's best gizmos. Empowering generations of gizmo making productive members of socienty, we continue a tradition of excellence and pure awsomeness. As we continue to venture into the 21st century, Gizmatic will continue to bring you the very best gizmos.</p>
+    </body>
+</html>
```

Comment:
Always good practice leave a newline at the end of all files! Your editor should have an option to add it automatically if it's missing :smile: